### DISCUSSION TITLE
node.cloudprovider.kubernetes.io/uninitialized=true:NoSchedule

### DISCUSSION THREAD

**Discussion by lovebiubiu** [k3s-io/k3s#6329] [Q&A]
Environmental Info:
k3s version:
k3s version v1.21.1+k3s1 (75dba57f)
go version go1.16.4
database:
oceanbase
cloud server:
aliyun ECS
Cluster Configuration:
1 server, 0 agents
Linux m001.cluster 3.10.0-1160.76.1.el7.x86_64 #1 SMP Wed Aug 10 16:21:17 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
Describe the bug:
steps:
run install.sh (from https://get.k3s.io/)

Expected behavior:
Cluster initializes and the service starts

Actual behavior:
node uninitialized with taints
![image](https://user-images.githubusercontent.com/32425177/197668703-ce116368-abb7-4c22-bc51-bf2efe90848d.png)

This is the only case where this problem occurs on aliyun server and oceanbase.My other environments installed successfully.

**Comment by brandond** [comment]
This indicates that the cloud controller has failed to initialize the node. Did you by any chance start k3s with `--disable-cloud-controller` or put `disable-cloud-controller: true` in the config file? If not, then please attach the full k3s-server log from journald.

Also: v1.21.1+k3s1 is end of support. Please install a supported release (v1.23-v1.25).
